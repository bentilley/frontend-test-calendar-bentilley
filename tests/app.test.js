import React from 'react';
import renderer from 'react-test-renderer';

import App from 'components/app/app';

jest.mock('components/calendar/calendar', () => 'calendar');

let component;

describe('App component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<App />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
});
