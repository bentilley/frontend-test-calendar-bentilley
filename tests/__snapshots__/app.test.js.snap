// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<main>
  <div
    id="header"
  >
    <h1>
      Wicked
    </h1>
    <p>
      
    </p>
  </div>
  <div
    className="main-content"
  >
    <div
      id="calendar-container"
    >
      <NumTickets
        numTickets={0}
        updateTicketNumber={[Function]}
      />
      <calendar
        eventId="2GXJ"
        selectedPerf=""
        show={0}
        updatePerfId={[Function]}
      />
    </div>
    <div
      id="plan-container"
    >
      <SeatingPlan
        eventId="2GXJ"
        selectedPerf=""
      />
    </div>
  </div>
</main>
`;
